resource_transip_private_network.go

```diff
@@ -0,0 +1,128 @@
+package main
+
+import (
+	"fmt"
+	"log"
+
+	"github.com/hashicorp/terraform/helper/resource"
+	"github.com/hashicorp/terraform/helper/schema"
+	"github.com/transip/gotransip/v6"
+	"github.com/transip/gotransip/v6/repository"
+	"github.com/transip/gotransip/v6/vps"
+)
+
+func resourcePrivateNetwork() *schema.Resource {
+	return &schema.Resource{
+		Create: resourcePrivateNetworkCreate,
+		Read:   resourcePrivateNetworkRead,
+		// Update: resourcePrivateNetworkUpdate,
```

Comment:
Does updating a private network make sense? As in, changing an attribute like description, without having to recreate the resource? In that case we could implement the update.

Reply:
I agree, it should be implemented as you may change your mind on the description :)